svc.html

```diff
@@ -0,0 +1,347 @@
+    enable user agents to support scalable video coding (SVC).</p>
+    <p>For Scalable Video Coding (SVC), the terms single-session transmission
+    (<dfn>SST</dfn>) and multi-session transmission (<dfn>MST</dfn>) are defined in
+    [[RFC6190]]. This specification only supports <a>SST</a> but not <a>MST</a>. The
+    term Single Real-time transport protocol stream Single Transport (<dfn>SRST</dfn>),
+    defined in [[RFC7656]] Section 3.7, refers to Scalable Video Coding
+    (<dfn>SVC</dfn>) implementations that transmit all layers within
+    a single transport, using a single Real-time Transport Protocol (RTP) stream
+    and synchronization source (SSRC). The term Multiple RTP stream Single Transport
+    (<dfn>MRST</dfn>), also defined in [[RFC7656]] Section 3.7, refers to
+    implementations that transmit all layers within a single transport, using
+    multiple RTP streams with a distinct SSRC for each layer. This specification
+    only supports <a>SRST</a> transport.</p>
+    <p>Within this specification it is assumed that scalable video coding operation is primarily
+    controlled utilizing the <code>addTransceiver</code> method of the <code><a>RTCPeerConnection</a></code>
+    and the <code>setParameters</code> method of the <code><a>RTCRtpSender</a></code> object.</p>
+    <p>The <code>addTransceiver</code> method establishes the <dfn>SVC envelope</dfn> which includes
+    the maximum number of scalable video coding layers that can be sent, as well as the ordering of the <code>encodings</code>.
+    While characteristics of individual scalable video coding layers can be modified using the <code>setParameters</code>
+    method, the <a>SVC envelope</a> cannot be changed.  One of the implications of this model is that
+    the <code>addTrack</code> method is not suitable for use with scalable video coding since this method
+    does not take <code>sendEncodings</code> as an argument, and therefore will not result in
+    configuration of an <code><a>RTCRtpTransceiver</a></code> with an <a>SVC envelope</a>.</p>
+    <p>Note that the inability to modify the <a>SVC envelope</a> using <code>setParameters</code> does not restrict
+    application flexibility in controlling the number of scalable video coding layers that are sent, or the characteristics of
+    those layers. For example, using <code>setParameters</code> individual SVC layers can be made inactive by setting the <code>active</code>
+    attribute to <code>false</code>, or reactivated by setting the <code>active</code>
+    attribute to <code>true</code>.</p> 
+    <p>This specification does not define how to configure <code>createOffer</code> to
+    receive scalable video coding. Codecs such as VP8, VP9 or AV1 require a compliant decoder
+    to be able to decode anything that an encoder can send so that configuration is not required
+    to enable an <code><a>RTCRtpReceiver</a></code> to decode scalable video coding.</p> 
```

Comment:
Will this be revisited?

Reply:
The assumption that a "decoder can decode anything an encoder can send" is valid for VP8 and VP9. It also appears to be valid for AV1, since support for SVC tools is required at all profile levels.